src/components/FormDatePicker/FormDatePicker.js

```diff
@@ -42,19 +43,34 @@ export function FormDatePicker(props) {
 
   const language = lang === "en" ? EN : lang === "fr" ? FR : EN;
 
-  const monthValues = [
-    "1",
-    "2",
-    "3",
-    "4",
-    "5",
-    "6",
-    "7",
-    "8",
-    "9",
-    "10",
-    "11",
-    "12",
+  const monthValuesEN = [
+    { id: "1", value: "January" },
+    { id: "2", value: "February" },
+    { id: "3", value: "March" },
+    { id: "4", value: "April" },
+    { id: "5", value: "May" },
+    { id: "6", value: "June" },
+    { id: "7", value: "July" },
+    { id: "8", value: "August" },
+    { id: "9", value: "September" },
+    { id: "10", value: "October" },
+    { id: "11", value: "November" },
+    { id: "12", value: "December" },
+  ];
+
+  const monthValuesFR = [
+    { id: "1", value: "janvier" },
```

Comment:
In French, months and days of the week are only lowercase if they're in the middle of a sentence.